pkg/utils/utils.go

```diff
@@ -14,7 +14,16 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-func ClusterName() string {
+func ClusterName(a, b string) string {
```

Comment:
@errordeveloper could you add a comment describing the logic in here, why you return string empty when both values are not nil?

Reply:
👍